mintscript/src/script/mod.rs

```diff
@@ -122,12 +122,25 @@ impl Threshold {
     }
 }
 
+#[derive(Clone, PartialEq, Eq, Debug)]
+pub enum HashChallenge {
+    RIPEMD160([u8; 20]),
+    SHA1([u8; 20]),
+    SHA256([u8; 32]),
+    HASH160([u8; 20]),
+    HASH256([u8; 32]),
```

Comment:
I don't think these need to be capitalized like this. Hash160, Sha256, etc, is OK.